### PR TITLE
Fix TiCS action's permission error

### PR DESCRIPTION
**Pull request by eaudetcobello**
`make go.unit` has to be run with elevated privileges since we now change the owner of /opt/cni/bin to root. See https://github.com/canonical/k8s-snap/blob/a2c00fb6be9951f582e139d83d97ad1dceb4ec13/src/k8s/pkg/k8sd/setup/directories.go#L72 

This operation fails if `make go.unit` is run as a regular user.

### CODE REVIEW

File: .github/workflows/cron-jobs.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 name: Security and quality nightly scan
 
 on:
-  schedule:
-    - cron: '0 10 * * *'
+  pull_request:
+  # schedule:
+  #   - cron: '0 10 * * *'
```
Comment (addyess) [inline .github/workflows/cron-jobs.yaml:6]: obviously revert this

**Review by addyess — CHANGES_REQUESTED** [review]
Looks really good.  Don't forget the workflow changes

File: .github/workflows/cron-jobs.yaml (hunk 1 above)
Comment (eaudetcobello) [reply .github/workflows/cron-jobs.yaml:6]: There's a new problem now: https://github.com/canonical/k8s-snap/actions/runs/11019106186/job/30604131753?pr=694#step:7:7502

**Review by addyess — APPROVED** [review]
Here's your 👍🏻 take it!

**Pull request merged**